eng/vscode/README.md

```diff
@@ -4,7 +4,83 @@ Easily bring the power of Model Context Protocol (MCP) to your Azure projects in
 
 ## Overview
 
-**Azure MCP Server** adds smart, context-aware AI tools right inside VS Code to help you work more efficiently with Azure resources.
+**Azure MCP Server** adds smart, context-aware AI tools right inside VS Code to help you work more efficiently with Azure resources. The Azure MCP Server supercharges your agents with Azure context across **28 different Azure services**.
+
+## ✨ What can you do with the Azure MCP Server?
```

Comment:
I suspect this will push the getting started section down below where user will have to scroll to. 
We should add a ToC section.

Reply:
Do we want to hold on this PR and let you do it better?  I'm ok either way.

Reply:
I am pushing my changes to this PR.